### ISSUE TITLE
20.04 branch missing 'unzip'

### ISSUE THREAD

**Issue by virtuald** [mstorsjo/msvc-wine#14]

**Comment by virtuald** [comment]
Ah, the reason I ran into this was most likely because I was trying to make the image smaller by `--skip-recommended`, but that seems to have broken the entire install.

**Comment by mstorsjo** [comment]
Yes, `--skip-recommended` is hard to use with wine - I've tried to minimize the install with it, but before you get wine back into working condition, you essentially have used up all the space you saved anyway.
